pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -674,10 +678,12 @@ class Measurements final
             });
 
         // Sampling using Random_XorShift64_Pool
-        Kokkos::Random_XorShift64_Pool<> rand_pool(
-            std::chrono::high_resolution_clock::now()
-                .time_since_epoch()
-                .count());
+        Kokkos::Random_XorShift64_Pool<> rand_pool =
```

Comment:
can we use `auto` to replace `Kokkos::Random_XorShift64_Pool<>` here?

Reply:
Good suggestion, since now the type is specified in the initializing values!